htdocs/components/00_jquery_validate.js

```diff
@@ -123,11 +123,13 @@ $.extend($.validator, {
     validClass:    'valid',
     invalidClass:  'invalid',
     requiredClass: 'required',
-    trim: [ 'int', 'nonnegint', 'posint', 'float', 'nonnegfloat', 'posfloat', 'email', 'url' ],
+    trim: [ 'int', 'nonnegint', 'posint', 'float', 'nonnegfloat', 'posfloat', 'email', 'url','name','required' ],
     tests: {
       'int':    new RegExp(/^[\-+]?\d+$/),
       'float':  new RegExp(/^([\-+]?)(?=\d|\.\d)\d*(\.\d*)?([Ee]([\-+]?\d+))?$/),
       password: new RegExp(/^\S{6,32}$/),
+      required: new RegExp(/(.|\s)*\S(.|\s)*/),
```

Comment:
We shouldn't need this field. Tests for required fields were passing even before this PR.